sso-kit-starter-hilla/src/main/resources/META-INF/spring/org.springframework.boot.autoconfigure.AutoConfiguration.imports

```diff
@@ -0,0 +1,2 @@
+dev.hilla.sso.starter.SingleSignOnDefaultBeans
+dev.hilla.sso.starter.SingleSignOnConfiguration
```

Comment:
I had to add `dev.hilla.sso.starter.SingleSignOnProperties` to the list to be able to run the Hilla demo application. This is due to different annotations in `dev.hilla.sso.starter.SingleSignOnConfiguration` compared to the analog class in Flow Kit

Reply:
I guess it's better to uniform the `*Configuration` classes.